src/v/kafka/server/server.cc

```diff
@@ -213,10 +213,10 @@ ss::future<> server::apply(ss::lw_shared_ptr<net::connection> conn) {
       mtls_state,
       config::shard_local_cfg().kafka_request_max_bytes.bind());
 
-    co_return co_await ss::do_until(
-      [ctx] { return ctx->is_finished_parsing(); },
```

Comment:
nit: `is_finished_parsing()` can be `private` now

Reply:
fixed can ya reapprove?